ui/src/app/base/components/NotificationList/NotificationList.tsx

```diff
@@ -30,19 +34,19 @@ const NotificationList = (): JSX.Element => {
   const notifications = {
     warnings: {
       items: useSelector(notificationSelectors.warnings),
-      type: "caution",
+      severity: NotificationSeverity.CAUTION,
```

Comment:
Yeah we don't have to, but I thought since this isn't actually interacting with the `Notification` component directly it might be good to use the const, so we know what it's meant for.